### DISCUSSION TITLE
An Area Calculations Tool !

### DISCUSSION THREAD

**Discussion by BigUV** [grafana/grafana#38385] [Dashboards, Panels & Visualizations]
Is there a tool for calculating an area marked on a graph depression ?
If not, it would be an amazing most handy tool, and not so complicated to develop.
Basically, it should calculate the area below the mark, above it, or even with full free hand drawn one.

[**Why is this needed**:](https://community.grafana.com/t/tool-for-calculating-depression-sum/31376)

**Comment by BigUV** [comment]
Please promote it!

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
